tests/ReferenceSQLTest.php

```diff
@@ -51,12 +51,12 @@ public function testBasic()
         $this->assertNull($oo->get('amount'));
 
         $oo = $u->unload()->addCondition('id', '>', '1')->ref('Orders');
-        if ($this->driverType === 'sqlite') {
-            $this->assertSame(
-                'select "id","amount","user_id" from "order" where "user_id" in (select "id" from "user" where "id" > :a)',
-                $oo->action('select')->render()
-            );
-        }
+
+        $e = $this->getEscapeChar();
+        $this->assertSame(
+            str_replace('"', $e, 'select "id","amount","user_id" from "order" where "user_id" in (select "id" from "user" where "id" > :a)'),
```

Comment:
better to put `$e` / expr. there directly

Reply:
That will make very long and unreadable SQL string.